### DISCUSSION TITLE
First Party Support for Emacs

### DISCUSSION THREAD

**Discussion by rwilson-lib** [community/community#51001] [Copilot Conversations]
### Select Topic Area

Question

### Body

So us some love Microsoft and add Emacs to the Officially Support IDE. I use Emacs for everything.

**Comment by iamcjwilliams** [comment]
@rwilson-lib Thank you for your feedback 🚀

**Comment by Remillard** [comment]
I also require official support for Emacs for use with the GitHub Enterprise Copilot.  The organization purchases support but only a handful of clients are supported.  I would greatly appreciate a supported plugin for this service for Emacs as this is my primary design tool.
